infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/ValidatorLogger.java

```diff
@@ -73,11 +74,19 @@ public void dutyFailed(
       final UInt64 slot,
       final Set<String> maybeKey,
       final Throwable error) {
-    final String errorString =
-        String.format(
-            "%sFailed to produce %s  Slot: %s%s",
-            PREFIX, producedType, slot, formatValidators(maybeKey));
-    log.error(ColorConsolePrinter.print(errorString, Color.RED), error);
+    if (!Throwables.getRootCause(error).getMessage().contains("signer is no longer active")) {
```

Comment:
I'm not keen on using the message as the identifying feature here.  Didn't we use a custom exception type? So we could use `ExceptionUtil.getCause(error, SignerNotActiveException.class)` to identify these cases.

Reply:
yep the SignerNotActiveException is a bit annoying where it lives, I can look at moving that to somewhere i can access...